### PR TITLE
Drop version column

### PR DESCRIPTION
**Pull request by alokmenghrajani**
- [x] @mcpherrinm

### CODE REVIEW

File: server/src/test/java/keywhiz/service/daos/SecretContentDAOTest.java [hunk 1]
```diff
@@ -73,7 +72,7 @@ public void setUp() throws Exception {
 
   @Test public void createSecretContent() {
     int before = tableSize();
-    secretContentDAO.createSecretContent(secretContent1.secretSeriesId(), "encrypted", "creator",
+    secretContentDAO.createSecretContent(secretContent1.secretSeriesId()+1, "encrypted", "creator",
```
Comment (mcpherrinm) [inline server/src/test/java/keywhiz/service/daos/SecretContentDAOTest.java]: what's the +1 about?

Comment (alokmenghrajani) [inline server/src/test/java/keywhiz/service/daos/SecretContentDAOTest.java]: We can't have duplicates anymore.

Comment (mcpherrinm) [inline server/src/test/java/keywhiz/service/daos/SecretContentDAOTest.java]: makes sense.  r+

**Comment by mcpherrinm** [comment]
🚢

**Pull request merged**